api/envoy/api/v2/cds.proto

```diff
@@ -645,6 +654,50 @@ message Cluster {
   // outgoing connections made by the cluster. Order matters as the filters are
   // processed sequentially as connection events happen.
   repeated cluster.Filter filters = 40;
+
+  // [#not-implemented-hide:] New mechanism for LB policy configuration.
+  // Used only if the :ref:`lb_policy<envoy_api_field_Cluster.lb_policy>`
+  // field has the value :ref:`LOAD_BALANCING_POLICY_CONFIG
+  // <envoy_api_enum_value_Cluster.LbPolicy.LOAD_BALANCING_POLICY_CONFIG>`.
+  LoadBalancingPolicy load_balancing_policy = 41;
+}
+
+// New Load balancing policy selection.
+//
+// Every LB policy defined via this mechanism will be identified via a
```

Comment:
One other question: should we tag this as experimental with https://github.com/envoyproxy/envoy/blob/master/tools/protodoc/protodoc.py#L55, providing freedom to do backwards compat breaking changes while you iterate on this with gRPC, or are you highly confident we're good to go with what is here as the stable version for v2 at least?

Reply:
I think the probability of needing changes here is fairly low, but I agree that it doesn't hurt to call it experimental for now.

Done.